resources/functionMap.php

```diff
@@ -6414,7 +6414,7 @@
 'Memcache::decrement' => ['int', 'key'=>'string', 'value='=>'int'],
 'Memcache::delete' => ['bool', 'key'=>'string', 'timeout='=>'int'],
 'Memcache::flush' => ['bool'],
-'Memcache::get' => ['string|array|false', 'key'=>'string', '&flags='=>'array', '&keys='=>'array'],
+'Memcache::get' => ['string|array|false', 'key'=>'string|array', '&flags='=>'int|array'],
```

Comment:
Could we get more precise, e.g. string[] ?

Reply:
@staabm Good idea. Just added commit with string[] for key and int[] for flags